bin/ch-checkns.c

```diff
@@ -62,12 +61,14 @@ Example:\n\
   ok\n\
 \n";
 
-#define TRY(x) if (x) fatal_errno(__LINE__)
+#define TRY(x) if (!(x)) fatal_(__FILE__, __LINE__, errno, #x)
```

Comment:
Why reverse the sense of this macro? It seems to make the calls noisier.

Reply:
I initially forgot that C's `if(0)` is `false`. I wanted the fatal function to execute when the syscalls return failure (non-zero). 

I personally think `TRY (0 == syscall())` is clearer than `TRY (!syscall())`